eb

```diff
@@ -80,7 +80,7 @@ for python_cmd in ${EB_PYTHON} ${EB_INSTALLPYTHON} 'python' 'python3' 'python2';
 
         if [ ! -z $PYTHON ]; then
             # check whether easybuild.main is available for selected python command
-            $PYTHON -c "import $EASYBUILD_MAIN" 2> /dev/null
+            $PYTHON -c "import easybuild" 2> /dev/null
             if [ $? -eq 0 ]; then
                 verbose "'$python_cmd' is able to import '$EASYBUILD_MAIN', so retaining it"
```

Comment:
Would need to change this output too to be consistent.

Reply:
done